From 68a873bd05a0e4271a87a818f26640cb579c3d27 Mon Sep 17 00:00:00 2001
From: Con Kolivas <kernel@kolivas.org>
Date: Tue, 5 May 2020 15:12:35 +1000
Subject: [PATCH 16/19] Disable forced context tracking with muqss which is
 seen as high levels of system load when idle since we are in kernel context
 but looks wrong in cpu load measurements to userspace, which is a problem
 when it is enabled by default when full dynticks is enabled.

---
 kernel/time/Kconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/kernel/time/Kconfig b/kernel/time/Kconfig
index fcc42353f125..60b4fd254c80 100644
--- a/kernel/time/Kconfig
+++ b/kernel/time/Kconfig
@@ -123,7 +123,7 @@ config CONTEXT_TRACKING
 
 config CONTEXT_TRACKING_FORCE
 	bool "Force context tracking"
-	depends on CONTEXT_TRACKING
+	depends on CONTEXT_TRACKING && !SCHED_MUQSS
 	default y if !NO_HZ_FULL
 	help
 	  The major pre-requirement for full dynticks to work is to
-- 
2.25.1

